package/MDAnalysis/analysis/encore/similarity.py

```diff
@@ -176,7 +176,7 @@
 import logging
 
 import numpy as np
-import scipy.stats
+from scipy.stats import gaussian_kde
```

Comment:
Please don't. This looses all the context and when browsing code it is not all clear where this function comes from. Is it defined locally? Is it imported from elsewhere?

I really hate it when used for things not in the standard library.

Please revert.